components/atom/backToTop/src/index.scss

```diff
@@ -0,0 +1,82 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$z-back-to-top: 10000;
```

Comment:
maybe you could use on the new z-index vars

Reply:
Yes, but which one?
```
$z-spinner: 1500 !default;
$z-cmp: 1400 !default;
$z-banner: 1300 !default;
$z-notifications: 1200 !default;
$z-modal: 1100 !default;
$z-sidebar: 1000 !default;
$z-tooltips: 900 !default;
$z-navigation: 800 !default;
```

Maybe `$z-tooltips` ?